Models/ObjectsFactoryModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EntityManager.Models
+{
+    public class ObjectsFactoryModel
+    {
+        public string VbName { get; set; }
+        public string Description { get; set; }
+        public List<FunctionModel> Functions { get; set; } = new List<FunctionModel>();
```

Comment:
maybe you should use IEnumerable<T> or ICollection<T> -> you do not need whole list implementation.<br>
you are using .Add function below then ICollection would be better -> all depends on usage.<br>

Reply:
Changed to ICollection